templates/frontend/pages/indexJournal.tpl

```diff
@@ -66,6 +66,7 @@
 									{$announcement->getLocalizedTitle()|escape}
 								</a>
 							</h4>
+							<p>{$announcement->getLocalizedDescriptionShort()}</p>
```

Comment:
I'd escape this with `strip_unsafe_html`

Reply:
Done